charts/kubewarden-controller/README.md

```diff
@@ -14,15 +14,20 @@ it is deployed.
 
 ## Installation
 
-The kubewarden-controller can be deployed using a helm chart:
-
-```shell
+The kubewarden-controller can be deployed using a helm chart.
+To install the kubewarden-controller in an existing cluster, make sure you have
+[`cert-manager` installed](https://cert-manager.io/docs/installation/) and then
+install the kubewarden-controller.
+
+For example:
+```console
+$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
```

Comment:
we can add a link to cert-manager documentation as there are more ways to install it https://cert-manager.io/docs/installation/

Reply:
The link is already present: https://github.com/kubewarden/helm-charts/pull/34/files#diff-c4a9c94dc08b8d5dc7ef6b4d15dc9a2486aeff530651d0eb58d87c9b8725e61dR19

Reply:
ah sorry I missed it